File: sth/finance_sth/report/laporan_histori_aktual_pdo/laporan_histori_aktual_pdo.py
# Copyright (c) 2026, DAS and contributors
# For license information, please see license.txt

from frappe import _

def execute(filters=None):
	conditions = get_condition(filters)
	columns = get_columns(filters)
	data = []

	data.append({
		"no_transaksi": "ACC-PAY-2026-00082"
	})
 
	return columns, data

def get_condition(filters):
	conditions = ""
	return conditions

def get_columns(filters):
	columns = [
		{
			"label": _("No Transaksi"),
			"fieldtype": "Data",
			"fieldname": "no_transaksi",
		},
		{
			"label": _("Tanggal"),
			"fieldtype": "Date",
			"fieldname": "tanggal",
		},
		{
			"label": _("Keterangan"),
			"fieldtype": "Data",
			"fieldname": "keterangan",
		},
		{
			"label": _("Jumlah"),
			"fieldtype": "Currency",
			"fieldname": "jumlah",
		},
	]

	return columns

File: sth/finance_sth/report/laporan_histori_aktual_pdo/laporan_histori_aktual_pdo.js
// Copyright (c) 2026, DAS and contributors
// For license information, please see license.txt

frappe.query_reports["Laporan Histori Aktual PDO"] = {
	"filters": [
		{
			"fieldname": "row_id",
			"label": __("Row ID"),
			"fieldtype": "Data",
		},
		{
			"fieldname": "type",
			"label": __("Type"),
			"fieldtype": "Select",
			"options": [
				"Bahan Bakar",
				"Perjalanan Dinas",
				"Kas",
				"Dana Cadangan",
				"Non PDO",
			],
		},
	]
};
